CalculatorKotlin/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,185 @@
+package org.poach3r
+
+import org.gnome.adw.Application
+import org.gnome.adw.ApplicationWindow
+import org.gnome.gdk.Display
+import org.gnome.gdk.Gdk
+import org.gnome.gdk.ModifierType
+import org.gnome.gio.ApplicationFlags
+import org.gnome.gtk.*
+import kotlin.system.exitProcess
+
+fun main(args: Array<String>) {
+    App().run(args)
+}
+
+class App: Application("org.poacher.GtkLayerShellTest", ApplicationFlags.DEFAULT_FLAGS) {
+    init {
+        onActivate {
+            loadCSS()
+            Window(this).present()
+        }
+    }
+
+    private fun loadCSS() {
+        val provider = CssProvider()
+        provider.loadFromPath("src/main/resources/Calculator.css")
+
+        StyleContext.addProviderForDisplay(
```

Comment:
Change this to `Gtk.styleContextAddProviderForDisplay` to prevent a deprecation warning about the `StyleContext` class.